docs/guides/snowflake_integration.md

```diff
@@ -486,3 +505,128 @@ FN_T_UID2_IDENTITY_MAP(EMAIL_HASH, 'email_hash')
 When you have the new function implemented, you can check the `UNMAPPED` column returned by the `FN_T_UID2_IDENTITY_MAP`. If any DII could not be mapped to a UID2, this column gives the reason.
 
 For details about the values and their explanations, see [Values for the UNMAPPED Column](#values-for-the-unmapped-column).
+
+## Usage for UID2 Sharers
```

Comment:
I think `FN_UID2_ENCRYPT` and `FN_UID2_DECRYPT` should be documented under "Shared Objects".

Reply:
That's a tricky one... we paralleled the way it was done for the server-side SDKs. But, those are lacking in action function reference as yet. Pending a question.

Reply:
@mcollins-ttd : See my update on this... referenced them in the Shared Objects section, as discussed.